src/parser/jobs/vpr/modules/SerpentsTail.tsx

```diff
@@ -4,11 +4,16 @@ import {Event} from 'event'
 import {Analyser} from 'parser/core/Analyser'
 import {filter, oneOf} from 'parser/core/filter'
 import {dependency} from 'parser/core/Injectable'
-import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import Checklist, {Requirement, TARGET, TieredRule} from 'parser/core/modules/Checklist'
 import {Data} from 'parser/core/modules/Data'
 import React from 'react'
 import DISPLAY_ORDER from './DISPLAY_ORDER'
 
+const TARGET_TIERS = {
+	99: TARGET.WARN,
+	100: TARGET.SUCCESS,
+}
+
 export class SeprentsTail extends Analyser {
 	static override handle = 'SeprentsTail'
```

Comment:
nitpick: handle should match the class & file name
```suggestion
	static override handle = 'SerpentsTail'
```